src/frontend/tests/pui_printing.spec.ts

```diff
@@ -84,6 +84,15 @@ test('PUI - Report Printing', async ({ page }) => {
 test('PUI - Report Editing', async ({ page }) => {
   await doQuickLogin(page, 'admin', 'inventree');
 
+  // activate the sample plugin for this test
+  await page.request.patch(`${classicUrl}/api/plugins/sampleui/activate/`, {
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Basic YWRtaW46aW52ZW50cmVl`
```

Comment:
Q: Could this be calculated/shared with the credentials in line 85 to ensure this does not break if we change default credentials?

Reply:
I have changed this now although the `admin:inventree` credentials are referenced already in a lot of different places.

Reply:
You're right, that makes sense.